bedrock/mozorg/views.py

```diff
@@ -335,26 +340,18 @@ def process_partnership_form(request, template, success_url_name, template_vars=
                 msg = 'ok'
                 stat = 200
             else:
-                # rename custom Salesforce fields to their real GUID name
-                data['00NU0000002pDJr'] = data.pop('interest')
-                data['00NU00000053D4G'] = data.pop('interested_countries')
-                data['00NU00000053D4L'] = data.pop('interested_languages')
-                data['00NU00000053D4a'] = data.pop('campaign_type')
-                data['oid'] = '00DU0000000IrgO'
                 data['lead_source'] = form_kwargs.get('lead_source',
                                                       'www.mozilla.org/about/partnerships/')
-                # As we're doing the Salesforce POST in the background here,
-                # `retURL` is never visited/seen by the user. I believe it
-                # is required by Salesforce though, so it should hang around
-                # as a placeholder (with a valid URL, just in case).
-                data['retURL'] = ('http://www.mozilla.org/en-US/about/'
-                                  'partnerships?success=1')
-
-                r = requests.post('https://www.salesforce.com/servlet/'
-                                  'servlet.WebToLead?encoding=UTF-8', data)
-                msg = requests.status_codes._codes.get(r.status_code, ['error'])[0]
-                stat = r.status_code
 
+                subject = PARTNERSHIPS_EMAIL_SUBJECT
+                sender = PARTNERSHIPS_EMAIL_FROM
+                to = PARTNERSHIPS_EMAIL_TO
+                msg = jingo.render_to_string(request, 'mozorg/emails/partnerships.txt', data)
```

Comment:
Looks like you should change this variable name. `msg` is used down on line 361 in the JSON value sent back to the client. I think you should make it 'ok' when everything is fine. But probably something like:

```python
try:
    email = EmailMessage(subject, msg, sender, to)
    email.send()
    stat = 200
    msg = 'ok'
except Exception:
    stat = 400
    msg = 'email send error'
```